### ISSUE TITLE
Needs documentation and examples

### ISSUE THREAD

**Issue by aapris** [josubg/pyfiware#1]
While RTFS is always an option, it would be nice to have at least basic documentation how to use the module and some examples for basic use cases (e.g for POST, PATCH, DELETE operations).

**Comment by josubg** [comment]
You are right. I put it in the # TODO List. meanwhile if you have any question Don't hesitate  to ask.

**Comment by YujiAzama** [comment]
Hi, @josubg 

I was looking for a useful client library in FIWARE and found pyfiware.
HistroyConnector class doesn't seem to be Orion's client library, what is it used for?
https://github.com/josubg/pyfiware/blob/master/pyfiware/history.py#L17
